app/src/main/java/org/simple/clinic/patient/Gender.kt

```diff
@@ -65,6 +65,14 @@ val Gender.displayLetterRes: Int
       is Unknown -> R.string.gender_unknown_letter
     }
 
+val Gender.displayChar : String
+  get() = when(this) {
+    Male -> "M"
+    Female -> "F"
+    Transgender -> "T"
+    is Unknown -> "U"
+  }
```

Comment:
We already have a `displayLetterRes` property which handles different languages. Why are we introducing a hard-coded english character string here?

Reply:
We don't. Removing it.